scripts/mkcloud

```diff
@@ -1084,8 +1084,6 @@ Optional
     scenario_dir='/tmp' (default=${SCRIPTS_DIR}/scenarios/cloud\$(getcloudver)/)
         Full path to directory containing scenario file.
 EOUSAGE
-    # Source qa_crowbarsetup.sh, which is where onadmin_help lives
-    source ${SCRIPTS_DIR}/qa_crowbarsetup.sh
```

Comment:
This commit breaks help, doesn't it? Could you squash this commit with the following one so that is doesn't?